### PR TITLE
BAMPFA-576: Added genres/themes vocabs to concepts and renamed associ…

### PR DESCRIPTION
**Pull request by cesarvh**
…ated to subjects

### CODE REVIEW

File: src/messages.js [hunk 1]
```diff
@@ -22,4 +22,7 @@ export default {
 
   'vocab.citation.local.name': 'Citations',
   'vocab.citation.collectionName.name': 'Citations',
+
+  'vocab.concept.associated.name': 'Subjects',
+  'vocab.concept.collectionName.name': 'Subjects',
 };
```
Comment (ray-lee) [inline src/messages.js]: I think you mean vocab.concept.associated.collectionName. Please make the name "Subject". As a general rule, name should be singular, collectionName plural. I didn't catch this with Citation, but same thing there.

File: src/plugins/recordTypes/concept/vocabularies.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { defineMessages } from 'react-intl';
+
+export default {
+  genre: {
+    messages: defineMessages({
+      name: {
+        id: 'vocab.concept.genre.name',
+        description: 'The name of the vocabulary.',
+        defaultMessage: 'Genres',
```
Comment (ray-lee) [inline src/plugins/recordTypes/concept/vocabularies.js]: Please make singular.

File: src/plugins/recordTypes/concept/vocabularies.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    serviceConfig: {
+      servicePath: 'urn:cspace:name(genre)',
+    },
+  },
+  theme: {
+    messages: defineMessages({
+      name: {
+        id: 'vocab.concept.theme.name',
+        description: 'The name of the vocabulary.',
+        defaultMessage: 'Themes',
```
Comment (ray-lee) [inline src/plugins/recordTypes/concept/vocabularies.js]: Please make singular.

**Review by ray-lee — CHANGES_REQUESTED** [review]
Hi Cesar, just a few changes to name messages. Also please make sure the JIRA issue stays accurate -- subject is a relabeling of associated (concept-concept), not an addition.

**Comment by cesarvh** [comment]
Pushed requested changes.

**Pull request merged**